pulsar-client-admin-shaded/pom.xml

```diff
@@ -66,6 +66,7 @@
                   <include>org.apache.pulsar:pulsar-client-original</include>
                   <include>org.apache.pulsar:pulsar-client-api</include>
                   <include>org.apache.pulsar:pulsar-client-admin-original</include>
+                  <include>org.apache.pulsar:pulsar-client-admin-api</include>
```

Comment:
Typically, the API module should be left out of the shaded jar, instead being a transitive dependency

Reply:
@merlimat `pulsar-client-api` is included as well.  The problem is with shading. If pulsar-client-admin-api is not included in the shaded JAR and is just pull in as a transitive dependency errors like

`java.lang.NoSuchMethodError: org.apache.pulsar.client.admin.PulsarAdminException.<init>(Lorg/apache/pulsar/shade/javax/ws/rs/ClientErrorException;)V`

will occur because the impl will be using the shaded version classes.

Reply:
I agree with @jerrypeng 
we already have` org.apache.pulsar:pulsar-client-api`is it an error ?